test/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ endforeach()
 ### Online check
 #
 #
-add_executable(onlinecheck onlinecheck.c ../src/config.c ../src/pgp.c)
+add_executable(onlinecheck onlinecheck.c ../src/config.c ../src/config_posix.c ../src/pgp.c)
 add_dependencies(onlinecheck src-onlinecheck)
```

Comment:
voorstel is dus om dit niet zo te doen maar met #include "config_posix.c"

Reply:
teruggedraaid